apps/web/components/FrontpageTabs/illustrations/Illustration.tsx

```diff
@@ -0,0 +1,420 @@
+import { theme } from '@island.is/island-ui/theme'
+import React, { FC } from 'react'
+import dynamic from 'next/dynamic'
+import { useWindowSize } from 'react-use'
+
+type IllustrationProps = {
+  illustrationIndex?: number
+}
+
+const illustrations = [
+  dynamic(() => import('./ManWithPhoneIllustration')),
+  dynamic(() => import('./WomanWithLaptopIllustration')),
+  dynamic(() => import('./WomanFeedingBabyIllustration')),
+  dynamic(() => import('./ManWithStrollerIllustration')),
+  dynamic(() => import('./WomanOnBusIllustration')),
+  dynamic(() => import('./ManOnBenchIllustration')),
```

Comment:
Are you going to be extracting these illustrations into another shared lib in a future PR?

Reply:
These illustrations won't be used in FrontpageTabs in the future. For other use cases we'll move them into island-ui/assets or similar, later.